packages/api-bindings/src/lens/utils/open-actions.ts

```diff
@@ -0,0 +1,209 @@
+/* eslint-disable no-case-declarations */
```

Comment:
I suspect this file was accidentally re-introduced while resolving a conflict.

Reply:
Removed.